### PR TITLE
Add Prettier and ESLint on web

### PR DESCRIPTION
**Pull request by Mevas**

### CODE REVIEW

File: .idea/.gitignore [hunk 1]
```diff
@@ -0,0 +1,8 @@
+# Default ignored files
```
Comment (adelinaenache) [inline .idea/.gitignore:1]: what?

File: .idea/MenuMaker.iml [hunk 2]
```diff
@@ -0,0 +1,12 @@
+<?xml version="1.0" encoding="UTF-8"?>
```
Comment (adelinaenache) [inline .idea/MenuMaker.iml:1]: I think that .gitignore is not working

File: frontend/.eslintrc.js [hunk 3]
```diff
@@ -0,0 +1,57 @@
...
+  },
+  overrides: [
+    {
+      files: ['**/*.tsx'],
+      rules: {
+        'react/prop-types': 'off',
+        'react-hooks/exhaustive-deps': [
+          'warn',
+          {
+            additionalHooks: 'useRecoilCallback',
```
Comment (adelinaenache) [inline frontend/.eslintrc.js:51]: NO

**Review by adelinaenache — CHANGES_REQUESTED** [review]

File: .idea/MenuMaker.iml (hunk 2 above)
Comment (Mevas) [reply .idea/MenuMaker.iml:1]: It is, those are global configuration files for anyone else who is using WebStorm.

File: .idea/.gitignore (hunk 1 above)
Comment (Mevas) [reply .idea/.gitignore:1]: https://www.toptal.com/developers/gitignore/api/webstorm

File: frontend/.eslintrc.js (hunk 3 above)
Comment (Mevas) [reply frontend/.eslintrc.js:51]: I'd rather deal with a warning than a hard to find bug.

**Pull request merged**